ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,25 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.1.4-rc1 27-Jan-2022
+Changes from 4.1.3
+1) Benewake CAN Lidar support
+2) CAN GPS default lag dropped to 0.1 seconds (was 0.22 seconds)
+3) Bug fixes
+    a) Compass custom orientation is never overwritten during calibration
+    b) EKF logging gaps fixed (some messages were occasionally being skipped)
+    c) Guided mode cornering improvements
+    d) IMU logging fix for IREG message (records IMU register changes)
+    e) LOITER_TO_ALT mission command's climb rate fixed (could climb or descend too quickly)
+    f) Position controller init fix to avoid twitch on vehicles with high vibrations
+    g) Position controller init fix to better handle high speed entry to flight mode
+    h) Position controller priortises reducing cross track error
```

Comment:
```suggestion
    h) Position controller prioritises reducing cross track error
```

Reply:
Oh, shoot, sorry, I saw this but then forgot to fix this before merging.